### PR TITLE
Update QSS3KeyPrefix parameter properties and fix glaring typos

### PR DESCRIPTION
**Pull request by carrollh**
You should be able to diff these without whitespace issues.

### CODE REVIEW

File: templates/sios-datakeeper-master.template [hunk 1]
```diff
@@ -801,15 +734,7 @@
...
-                        "Fn::Join": [
-                            "/",
-                            [
-                                {
-                                    "Ref": "QSS3KeyPrefix"
-                                },
-                                "submodules/quickstart-microsoft-activedirectory"
-                            ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-activedirectory"
```
Comment (santiagocardenas) [inline templates/sios-datakeeper-master.template]: Needs a slash at the end:
"Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-activedirectory/"

File: templates/sios-datakeeper-master.template [hunk 2]
```diff
@@ -898,15 +813,7 @@
...
-                        "Fn::Join": [
-                          	"/",
-                           [
-                             	{
-                                	"Ref": "QSS3KeyPrefix"
-                             	},
-                              	"submodules/quickstart-microsoft-rdgateway"
-                           ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway"
```
Comment (santiagocardenas) [inline templates/sios-datakeeper-master.template]: Needs a slash at the end:
"Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway/"

File: templates/sios-datakeeper-master.template [hunk 3]
```diff
@@ -831,27 +756,17 @@
...
-                                "Ref": "QSS3KeyPrefix"
-                            },
-                            "submodules/quickstart-microsoft-rdgateway",
-                            "templates/rdgw-domain.template"
-                        ]
+                            }
+                        }
                     ]
                 },
                 "Parameters": {
```
Comment (santiagocardenas) [inline templates/sios-datakeeper-master.template]: No longer need to pass
                     "ADServer1PrivateIP": {
                         "Ref": "ADServer1PrivateIP"
                     },
                     "ADServer2PrivateIP": {
                         "Ref": "ADServer2PrivateIP"
                     },

File: templates/sios-datakeeper-master.template [hunk 4]
```diff
@@ -898,15 +813,7 @@
...
-                             	{
-                                	"Ref": "QSS3KeyPrefix"
-                             	},
-                              	"submodules/quickstart-microsoft-rdgateway"
-                           ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway"
                     },
                     "RDGW1InstanceType": {
                         "Ref": "RDGW1InstanceType"
```
Comment (santiagocardenas) [inline templates/sios-datakeeper-master.template]: RDGW1InstanceType is now RDGWInstanceType.
RDGW2InstanceType, RDGW1 and RDGW2 PrivateIPs and NetBIOS names are no longer parameters.

Please check out the latest in the RDGW. We made those changes there after popular demand to decrease from 2 instances to 1 by default but allow a way to deploy more.

Comment (santiagocardenas) [inline templates/sios-datakeeper-master.template]: Remember to also remove those parameters from this master template and also from the groups and labels

File: templates/sios-datakeeper-master.template (hunk 1 above)
Comment (carrollh) [inline templates/sios-datakeeper-master.template]: Fixed

File: templates/sios-datakeeper-master.template (hunk 2 above)
Comment (carrollh) [inline templates/sios-datakeeper-master.template]: Fixed

File: templates/sios-datakeeper-master.template (hunk 3 above)
Comment (carrollh) [inline templates/sios-datakeeper-master.template]: Fixed, testing.

File: templates/sios-datakeeper-master.template [hunk 5]
```diff
@@ -369,13 +349,13 @@
...
-            "Description": "S3 key prefix for the Quick Start assets. Quick Start key prefix can include numbers, lowercase letters, uppercase letters, hyphens (-), and forward slash (/). It cannot start or end with forward slash (/) because they are automatically appended.",
+            "AllowedPattern": "^[0-9a-zA-Z-/]*$",
+            "ConstraintDescription": "Quick Start key prefix can include numbers, lowercase letters, uppercase letters, hyphens (-), and forward slash (/).",
+            "Default": "sios/datakeeper/latest/",
+            "Description": "S3 key prefix for the Quick Start assets. Quick Start key prefix can include numbers, lowercase letters, uppercase letters, hyphens (-), and forward slash (/).",
             "Type": "String"
         },
-        "RDGW1InstanceType": {
+        "RDGWInstanceType": {
             "Description": "Amazon EC2 instance type for the first Remote Desktop Gateway instance",
```
Comment (santiagocardenas) [inline templates/sios-datakeeper-master.template]: You'll want to change this description to:
"Amazon EC2 instance type for the Remote Desktop Gateway instances"

**Pull request merged**